File: FRONTEND/appFrontEnd/src/app/interfaces/Libro.ts
export interface Libro {
    id?: number;
    nombreAutor?: string;
    titulo?: string;
    descripcion?: string;
    fechaPublicacion?: string;
    numeroEdicion?: number;
    urlImagenPortada?: string;
    urlImagenIndice?: string;
    urlPdfLibro: string;
}

File: BACKEND/src/main/java/com/microservicio/libreria/dto/Libro.java
package com.microservicio.libreria.dto;

import lombok.Data;

@Data
public class Libro {
    private Integer id;
    private String nombreAutor;
    private String titulo;
    private String descripcion;
    private String fechaPublicacion;
    private Integer numeroEdicion;
    private String urlImagenPortada;
    private String urlImagenIndice;
    private String urlPdfLibro;


}